src/components/Breadcrumb/index.tsx

```diff
@@ -32,7 +32,7 @@ function flatten([
 export function Breadcrumb({ relativePath, basePath }: IBreadcrumbProps) {
   const paths = relativePath.split("/")
   const breadcrumbItems = flatten([traversePaths(paths, basePath)])
-
+  let count = 0;
```

Comment:
perhaps a better name, like `foldersDepth` would be better?

Reply:
Yeah definitely, bad habit of mine